nautobot/extras/forms.py

```diff
@@ -270,6 +274,179 @@ def save(self, commit=True):
         return obj
 
 
+class RelationshipModelBulkEditFormMixin(BulkEditForm):
+    """Bulk-edit form mixin for models that support Relationships."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.obj_type = ContentType.objects.get_for_model(self.model)
```

Comment:
Should/could this be hoisted up to `BulkEditForm`?

Reply:
No, IMHO, since `BulkEditForm` has no direct use for `self.obj_type`.